### ISSUE TITLE
it would be lovely to be able to switch from one timeline to the other with a swipe.

### ISSUE THREAD

**Issue by MariaFeodora** [Dimillian/IceCubesApp#181]

**Comment by PauloCarrasco** [comment]
I agree. Changing timelines with an horizontal swipe would really improve the one hand usability of the app.

**Comment by karimlevallois** [comment]
Agreed, similar to how Ivory lets you switch between accounts by swiping the title bar. It would be great to be able to quickly switch between Home, Local Timeline and Tag Group with a swipe left or right.
